src/gui/properties/speedplotview.h

```diff
@@ -29,9 +30,13 @@
 #pragma once
 
 #ifndef Q_MOC_RUN
```

Comment:
IIRC, there were some conflicts with `moc` and `boost/circular_buffer.hpp` header, so it was placed under this guard. I don't see the point in adding these new headers under this guard too.
BTW, perhaps it is no longer required at all.

Reply:
I think this make moc bit faster,

Reply:
I don't think it's really supposed to be used for this purpose. At least I don't recall that Qt recommended enclosing all third-party inclusions in such a guard. Seems it's just exceptional case when moc conflicts with some headers.
@Chocobo1, have you to say something about it?

Reply:
I'm not familiar with it, probably best not relying on it whenever possible...

Reply:
Please leave only circular buffer include inside `#ifndef Q_MOC_RUN`.